keyboards/kprepublic/bm43hsrgb/readme.md

```diff
@@ -0,0 +1,18 @@
+# bm43hsrgb
+
+A 43% (kind of ANSI) mechanical keyboard.
+
+This firmware was originally taken from a kbfirmware.json and converted through [kbf_qmk_converter](https://noroadsleft.github.io/kbf_qmk_converter/). You may find the original `.json` files [here](https://drive.google.com/drive/folders/11DowBYrFN_uCNa9Q9bXwuMn91vmZYBcG).
+
+
+* Keyboard Maintainer: [bitstarr](https://github.com/bitstarr)
+* Hardware Supported: bm43hsrgb PCB
+* Hardware Availability: [KPRepublic](https://kprepublic.com/collections/bm43/products/bm43-rgb-43-keys-40-custom-mechanical-keyboard-pcb-programmed-via-qmk-firmware-with-rgb-bottom-underglow-hot-swapping-switch)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make bm43hsrgb:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Instructions on how to reset the board to bootloader mode should be included above this paragraph.